ci/uvx-base.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+source base.sh
+
+run_uvx() {
+    # Look up the version of the command using the spec function and run the command using uvx.
+    command_spec=$(spec $1 @)
```

Comment:
I don't get it, where does the `@` come from? This produces something like `package@v1.2.3` - is that the syntax for uvx?
Also note that the pypi page for uvx links to a `uvx2` repo, which is apparently renamed to `uvenv` to avoid naming collision with a new uv command

Reply:
uvx is a bit inconsistent with the package spec, it uses both @ and ==: `uvx --with package1==a.b.c package2@x.y.z`. Hence the need to pass the separator to the `spec` function.

Note that `uvx` is not a separate package but an alias for `uv tool run`.

Reply:
ouch, no wonder I couldn't understand the docs!